_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -110,11 +110,14 @@ This `canvas` element has an `aria-label` attribute that describes the image.
 
 ```html
 <html lang="en">
-	<canvas id="act" width="200" height="60" aria-label="ACT Rules!"></canvas>
+	<canvas id="logo" width="72" height="48" aria-label="W3C"></canvas>
```

Comment:
```suggestion
	<canvas id="logo" width="72" height="48" aria-label="W3C logo"></canvas>
```
since it's the logo image.

Reply:
The other images in this rule don't have this either. If you think that needs to change I'd rather do it in a separate PR. If you want us to do that consistently, would you mind opening an issue for this instead?